core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/samModels.scala

```diff
@@ -4,17 +4,24 @@ import ca.mrvisser.sealerate
 import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
-final case class RuntimeSamResourceId(resourceId: String) extends AnyVal
-final case class PersistentDiskSamResourceId(resourceId: String) extends AnyVal
-final case class ProjectSamResourceId(googleProject: GoogleProject) extends AnyVal
-final case class AppSamResourceId(resourceId: String) extends AnyVal
+sealed trait SamResourceId {
+  def resourceId: String
+  def asString: String = resourceId
+}
+
+final case class RuntimeSamResourceId(resourceId: String) extends SamResourceId
+final case class PersistentDiskSamResourceId(resourceId: String) extends SamResourceId
+final case class ProjectSamResourceId(googleProject: GoogleProject) extends SamResourceId {
+  override def resourceId: String = googleProject.value
+}
+final case class AppSamResourceId(resourceId: String) extends SamResourceId
 
 sealed trait SamResourceType extends Product with Serializable {
   def asString: String
 }
 object SamResourceType {
   final case object Project extends SamResourceType {
-    val asString = "billing-project"
+    val asString = "google-project" //TODO: verify, is this correct?
```

Comment:
This is what the ticket says. I looked through where this gets plugged into the request to sam, and verified against a valid SAM payload. Just wanted additional verification on this

Reply:
Yup, this is correct.. you can remove the TODO